diff --git a/node_modules/laravel-mix/src/components/Notifications.js b/node_modules/laravel-mix/src/components/Notifications.js
index c022b24..961113c 100644
--- a/node_modules/laravel-mix/src/components/Notifications.js
+++ b/node_modules/laravel-mix/src/components/Notifications.js
@@ -9,13 +9,13 @@ class Notifications extends AutomaticComponent {
             let WebpackNotifierPlugin = require('webpack-notifier');
 
             return new WebpackNotifierPlugin({
-                appID: 'Laravel Mix',
+                appID: 'Django Mix',
 
-                title: 'Laravel Mix',
+                title: 'Django Mix',
                 alwaysNotify: Config.notifications.onSuccess,
-                timeout: false,
+                timeout: 2,
                 hint: process.platform === 'linux' ? 'int:transient:1' : undefined,
-                contentImage: Mix.paths.root('node_modules/laravel-mix/icons/laravel.png')
+                contentImage: Mix.paths.root('django.png')
             });
         }
     }
